projects/plugins/jetpack/extensions/blocks/contact-form/transforms.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import { pickBy, identity } from 'lodash';
+import { createBlock } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+const getAttributeValue = ( tag, att, content ) => {
+	const doubleQuotes = content.match( new RegExp( `\\[${ tag }[^\\]]* ${ att }="([^"]*)"`, 'im' ) );
+	if ( doubleQuotes != null && doubleQuotes.length > 0 ) {
```

Comment:
is it intentional that we want to check for ! `undefined` or `null` here? given how infrequently we end up using `!=` instead of `!==` it does look odd 😄

Reply:
Yes, thanks. This was actually a copy and paste job :)

Reply:
I'm also not used to these new ligatures I'm trying out 😂 

<img width="207" alt="Screen Shot 2021-01-28 at 1 40 49 PM" src="https://user-images.githubusercontent.com/1464705/106202248-7ad4df00-616e-11eb-8be3-d8b981220957.png">